pages/common/openssl-dgst.md

```diff
@@ -0,0 +1,24 @@
+# openssl dgst
+
+> OpenSSL command to generate digest values and perform signature operations.
+> More information: <https://www.openssl.org/docs/man1.0.2/man1/dgst.html>.
+
+- Generate a SHA256 hash value of a file:
+
+`openssl dgst -sha256 -binary -out {{output_file}} {{input_file}}`
+
+- Sign a file using and RSA key:
+
+`openssl dgst -sign {{private_key_file}} -sha256 -sigopt rsa_padding_mode:pss -out {{output_file}} {{input_file}}`
```

Comment:
The argument options should be between `{{}}`:
```suggestion
`openssl dgst -sign {{private_key_file}} -sha256 -sigopt {{rsa_padding_mode}}:{{pss}} -out {{output_file}} {{input_file}}`
```

Reply:
The option parts belong together. I would rather use `{{rsa_padding_mode:pss}}` instead

Reply:
Is there another possible option for `rsa_padding_mode`? I'm not sure, the man page said `-sigopt nm:v`.

Reply:
Yes. You can also use `pkcs1` and `none` for signatures.

Reply:
So, `nm` is an argument option and `v` is another argument option.

Reply:
No. `nm`is the name of the option and `v` is its value.

Reply:
I think this example is fine as-is.